### ISSUE TITLE
Parallelisation of iterator

### ISSUE THREAD

**Issue by beirtipol** [khmarbaise/iterator-maven-plugin#42]
Would it be possible to add a 'parallel' flag to the iterator goal? I have a collection of 'environment.zip' assemblies which take a few seconds each to build. Being able to fork each iteration would be great

**Comment by khmarbaise** [comment]
First what about https://github.com/khmarbaise/multienv-maven-plugin/ ?

**Comment by beirtipol** [comment]
That project does look interesting. I'll dig in and see if it fits what I'm doing. Thanks Karl

The reason I'm so concious of speed (and looking forward to thread the execution) is that my team need to be able to build all environment zip artifacts and then unzip them locally for debugging multiple envs. It takes about 2 minutes to run the maven build which can be a bit painful when you're repeatedly changing build properties.

**Comment by khmarbaise** [comment]
2 Minutes Hm... I think the mult env plugin should run faster...but I have not tested that...Maybe you can give an example how your setup at the moment looks like....

**Comment by beirtipol** [comment]
Yeah, we unfortunately have a hell of a lot of files in there. I'll
certainly give your multienv project a go.

Also, thank you massively for the iterator plugin. It has saved me a huge
amount of copy pasted pom!
